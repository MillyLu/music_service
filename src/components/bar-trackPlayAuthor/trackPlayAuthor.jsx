import { useContext } from "react";
import { ThemeContext } from '../../ThemeProvider';
import * as Styled from './styles';


export function TrackPlayAuthor({data}){
    const {theme} = useContext(ThemeContext)
    return(
        <Styled.TrackPlayAuthor>
            <Styled.TrackPlayLinkAuthor theme={theme} >{data.author}</Styled.TrackPlayLinkAuthor>
        </Styled.TrackPlayAuthor>
    )
}