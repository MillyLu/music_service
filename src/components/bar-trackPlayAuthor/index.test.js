/* eslint-disable arrow-body-style */
/* eslint-disable react/function-component-definition */
/* eslint-disable react/jsx-props-no-spreading */
/* eslint-disable no-undef */
import * as ShallowRenderer from 'react-shallow-renderer';
import { TrackPlayAuthor } from './trackPlayAuthor';

const Wrapped = ({ children }) => {
  return <TrackPlayAuthor>{children}</TrackPlayAuthor>
}



describe('<TrackPlyAuthor/>', () => {
        it('should render successfully and match snapshot', () => {
            const shallowRenderer = ShallowRenderer.createRenderer();
            const renderer = shallowRenderer;
            renderer.render(<Wrapped>author</Wrapped>);
            const tree = renderer.getRenderOutput();
            expect(tree).toMatchSnapshot();
          })
      });