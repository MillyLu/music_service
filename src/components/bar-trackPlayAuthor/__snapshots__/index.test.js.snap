// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<TrackPlyAuthor/> should render successfully and match snapshot 1`] = `
<TrackPlayAuthor>
  author
</TrackPlayAuthor>
`;
